app/cmd/lintAndInstall.go

```diff
@@ -0,0 +1,72 @@
+// Copyright © 2018 The Helm Authors
```

Comment:
The copyright doesn't need the year. "Copyright The Helm Authors" is sufficient according to the CNCF.

Reply:
Fixed.